layouts/office_hours/single.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="{{ with .Site.LanguageCode }}{{ . }}{{ else }}en-US{{ end }}">
+    {{- partial "head.html" . -}}
+    <body>
+    {{- partial "preloader.html" . -}}
+    {{- partial "static-header.html" . -}}
+
+
+<!--{{ partial "page-title.html" . }}-->
+
+
+
+<div class="coding_section container" >
+    <h1 style="text-align: center;">{{.Title}}</h1>
+            {{ partial "header_divider.html" .}}
+    <div class="row justify-content-center" > 
+        <div class="content-section col-md-8">
+            {{.Content}}
+
+            <div class="d-flex">
+                {{ with .NextInSection }}
+                <div>
+                <a href="{{ .Permalink }}"><i class=" ti-arrow-left text-black"></i></a>
+                <a href="{{ .Permalink }}">{{ .Title }}</a>
+                </div>
+                {{ end }}
+
+                {{ with .PrevInSection }}
+                <div style="margin-left: auto;">
+
+                    <a href="{{ .Permalink }}">{{ .Title }}</a>
+                    <a href="{{ .Permalink }}"><i class=" ti-arrow-right text-black"></i></a>
+                </div>
+                {{ end }}
+            </div>
+            <div class="d-flex justify-content-around" >
+                <div class="text-center border border-dark  p-2">
```

Comment:
Switch these to using the cards from the theme. You can copy and past the html from `layouts/shortcodes/card.html` as a starting place.